ts/components/services/ServicesHomeButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { constNull } from "fp-ts/lib/function";
+import { connect } from "react-redux";
+import FooterWithButtons from "../ui/FooterWithButtons";
+import I18n from "../../i18n";
+import { GlobalState } from "../../store/reducers/types";
+import { Dispatch } from "../../store/actions/types";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const ServicesHomeButton = (props: Props): React.ReactElement => (
+  <FooterWithButtons
+    type={"TwoButtonsInlineHalf"}
+    leftButton={{
+      title: I18n.t("services.disableAll"),
+      onPress: props.enableAll,
+      bordered: true,
+      light: true
+    }}
+    rightButton={{
+      title: I18n.t("services.enableAll"),
+      onPress: props.disableAll,
+      bordered: true,
+      light: true
+    }}
+  />
+);
+
+const mapStateToProps = (_: GlobalState) => ({});
+
+const mapDispatchToProps = (_: Dispatch) => ({
+  // TODO Add the handlers when available
+  enableAll: constNull,
+  disableAll: constNull
```

Comment:
In the past, I used to throw `new Error("xyz not implemented")` for features available only in _testing_ and dev environments. That makes immediately clear what's missing and what's happening, whereas a non working button might let the developer figure out whether is a bug or wip.

Reply:
We usually use `constNull`  or `constUndefined` to keep temporary functions and a TODO to mark that the implementation is still a wip.

Anyway component is not rendered if the FF is not enabled, when we will officially release the functionality we may remove the check on FF (since this will not be a temporary feature)